es-app/src/views/ViewController.cpp

```diff
@@ -339,6 +339,20 @@ std::shared_ptr<IGameListView> ViewController::getGameListView(SystemData* syste
 	return view;
 }
 
+void ViewController::deleteGameListView(SystemData* system)
```

Comment:
Maybe. As I said, way too many refactorings while developing. I don't think I have any future use for this one. I did a code cleanup before committing, but this one must have slipped - thanks for catching it!